api/resources/config/config.go

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
I'd recommend moving the ConfigResource API endpoint to a separate PR, if you don't mind?